Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:buster AS builder
+
+# Create webapp directory
+RUN mkdir -p /app/webapp
+
+# Copy JSX source files
+COPY webapp/src /app/webapp/src
+COPY webapp/public /app/webapp/public
+COPY webapp/package*.json /app/webapp/
+
+WORKDIR /app/webapp/
+
+# Build up the frontend
+ARG REACT_APP_ENDPOINT
+ENV REACT_APP_ENDPOINT=$REACT_APP_ENDPOINT
+
+RUN npm install --only=prod
+RUN npm run build
+RUN mkdir -p /app/static/
+
+RUN cp -r build/* /app/static
+RUN mv /app/static/static/* /app/static/
+RUN rm -r /app/static/static
+
+
+FROM python:buster
+
+# Copy contents from builder
+COPY --from=builder /app /app
```

Comment:
We should only be copying the build output, this pulls in the entire `node_modules` folder.

Reply:
ecf6404 
Only copying static over now

Reply:
👍